hacktoberfest/connector-contributor-guide.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Connector Contributor Guide - Ballerina Hacktoberfest 2024
+description: This guide will walk you through the process of participating in the Ballerina Hacktoberfest connector event.
+keywords: ballerina, hacktoberfest, connector, contribution, guide, hackathon
+intro: This provides a step-by-step guide for contributing to the Ballerina Hacktoberfest connector event.
+---
+
+Welcome to the Ballerina Hacktoberfest connector event! We're excited to have you contribute to the Ballerina ecosystem. This guide will walk you through the process of participating in the hacktoberfest and contributing your connector. Please follow the instructions carefully to ensure your contribution is eligible for rewards.
```

Comment:
Few things:
1. In Welcome to the `Ballerina Hacktoberfest connector event` -> We better link Ballerina hacktoberfest page here.
2. `hacktoberfest` -> should be changed to `Hacktoberfest`
3. Link general hacktoberfest event